jekyll/_cci2/status-badges.md

```diff
@@ -21,19 +22,118 @@ You can generate code for the following formats:
 - reStructuredText
 - pod
 
-## Steps
+## Generating a status badge
 
-1. Locate the Settings page for your project by clicking on the "cog" icon at the top right of the Jobs page.
-2. In the _Notifications_ section of your project's settings, click _Status Badges_.
-3. By default, the badge displays the status of your project's default branch. If you want to show the status of a different branch, use the _Branch_ dropdown menu to select it.
-3. (Optional) If your project is private, you will need to [create a project API token]({{ site.baseurl }}/2.0/managing-api-tokens/#creating-a-project-api-token).
-4. (Optional) If you created a token in the previous step, select the token you want to use in the _API Token_ dropdown menu.
-5. Select the appropriate language from the _Embed Code_ dropdown menu.
-6. Copy and paste the generated link in the document where you want to display the status badge.
+To generate your own status badge, you will need to know and substitute the
+following variables in the code snippets below:
 
-![](  {{ site.baseurl }}/assets/img/docs/status_badge_embed.png)
+- `<PROJECT_NAME>` - Your project's name. Example: `circleci-docs`
+- `<ORG_NAME>` - The organization or user name the project in question belongs to.
+- `<VCS>` - your VCS provider (`gh` for "github" and `bb` for BitBucket).
+- `<LINK>` - The link you want the status badge to go to
+  when clicked. Most users will link to their CircleCI build's page.
+- optional: an API token (to create badges for private projects.)
```

Comment:
Is creating private project badges the only use case for adding an API token? if so probably don't need the parentheses, and where do they put the API token in the templates below?

Reply:
Yes and no; there are different API scopes that on can choose when creating a token; in this case the user needs to create a scoped "status" API token. I think I could simplify this by just saying "a status API token".